distributed/utils_test.py

```diff
@@ -112,6 +112,8 @@ def is_debugging() -> bool:
 _TEST_TIMEOUT = 30
 _offload_executor.submit(lambda: None).result()  # create thread during import
 
+NO_AMM = {"distributed.scheduler.active-memory-manager.start": False}
```

Comment:
nit:
```suggestion
NO_AMM_CONFIG = {"distributed.scheduler.active-memory-manager.start": False}
```

Reply:
I find it very redundant and verbose to see `config=NO_AMM_CONFIG` every time

Reply:
Maybe an alternative solution would be a TypeAlias to make it clearer that the `NO_AMM` constant is a configuration?

Reply:
Such a TypeAlias would need to go in dask/dask, and I wouldn't be able to enforce it in the signature of gen_cluster anyway.
Would it be OK if I just wrote an explanatory comment on the definition?

Reply:
Fair point, the TypeAlias doesn't warrant any hold-up on this PR. Thanks for adding a comment. Generally with nits, feel free to ignore them :)